web/themes/custom/novel/templates/views/views-view--events.html.twig

```diff
@@ -1,7 +1,13 @@
 {% extends '@novel/views/views-view--content-list-page.html.twig' %}
 
 {% block facets %}
-  {{ drupal_block("facet_block:event_categories") }}
-
-  {{ drupal_block("facet_block:branch") }}
+  <div class="content-list-page__filter">
+    {{ drupal_block("facet_block:event_categories") }}
+  </div>
+  <div class="content-list-page__filter">
+    {{ drupal_block("facet_block:branch") }}
+  </div>
+  <div class="content-list-page__filter">
+    {{ drupal_block("facet_block:date") }}
+  </div>
```

Comment:
Is this ment to be here? I see no mention of date filtering and as far as i can see this is not a facet block on `/admin/config/search/facets`

It's causing a lot of warnings on `/events` when logged in.

Reply:
You are quite right. I messed up the juggling between the date facet and the category and branch.